docs/installing.md

```diff
@@ -1,6 +1,161 @@
+
+You can do this on a local Rails console, by logging in and setting the boolean field 'admin'
+on your user ID to True.
+If you'd prefer to do this on the Heroku deployment, make sure you've logged into the application.
+Then you can directly modify this attribute in the deployments Postgres database using SQL.
+For more information on accessing your application's Postgres database,
+see [the official docs](https://devcenter.heroku.com/articles/heroku-postgresql#pg-psql).
+
+Making modifications to launch your own site
+--------------------------------------------
+
+Some times you may not want to launch an exact copy of JOSS, but a modified version.
+This can be especially useful if you're planning to spin up your own platform based on the
+Open Journals framework.
+[NeuroLibre](https://neurolibre.herokuapp.com) is one such example use-case.
+
+Modifying your site configuration
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+In this case, there are several important variables to be aware of and modify.
+Most of these are accessible in the `config` folder.
+
+First, there are three files which provide settings for your Rails application in different development contexts:
+
+1. `settings-test.yml`
+1. `settings-development.yml`
+1. `settings-production.yml`
+
+These each contain site-specific variables that should be modified if you are building off of the Open Journals framework.
+
+Next, you'll need to modify the `repository.yml` file.
+This file lists the GitHub repository where you expect papers to be published,
+as well as the editor team ID.
+For your GitHub organization, make sure you have created and populated a team called `editors`.
+Then, you can check its ID number as detailed in [this guide](https://fabian-kostadinov.github.io/2015/01/16/how-to-find-a-github-team-id).
+In `config` you should also modify the `orcid.yml` file to list your site as the production site.
+
+Optionally, you can edit `seeds.rb`, a file in the `db` folder.
+"DB" is short for "database," and this file _seeds_ the database with information about your editorial team.
+You can edit the file `seeds.rb` to remove any individuals who are not editors in your organization.
+This file will be updated later as you add information in the application, so this step is not strictly necessary.
+
+Modifying your site contents
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+You can modify site content by updating files in the `app` and `docs` folders.
+For example, in `app/views/notifications` you can text for any emails that will be sent by your application.
+
+Note that files which end in `.html.erb` are treated as HTML files, and typical HTML formatting applies.
+You can set the HTML styling by modifying the Sass files for your application,
+located in `app/assets/stylesheets`.
+
+There are currently a few hard-coded variables in the application which you will also need to update.
+Note that these are mostly under `lib/tasks`.
```

Comment:
@arfon I'm not sure if you'd like me to open new issues that these values are hard-coded ? At least for the google sheet info, I'd think that could go in the site config !

Reply:
Agreed. These should probably be extracted.

Reply:
Opened #826 !